test.disabled/ejabberd_tests/tests/mam_SUITE.erl

```diff
@@ -813,8 +815,7 @@ init_state(C, muc_light, Config) ->
 init_state(_C, prefs_cases, Config) ->
     Config;
 init_state(_C, policy_violation, Config) ->
-    rpc_apply(mod_mam, set_params,
-              [ [{max_result_limit, 5}] ]),
+    rpc_apply(gen_mod, set_module_opt, [host(), mod_mam, max_result_limit, 5]),
```

Comment:
can we put this call into a helper module?

Reply:
I prefer this to be explicit since there's some magic going on with external option setting.